File: Day-5/program2.cpp
#include<iostream>
using namespace std;
int main()
{//opening main
    int n,d,p=1,sum=0,num;//initialising variables
    cout<<"ENTER NUMBER";
    cin>>n;//input number from keyboard
    num=n;
    while(n!=0)
    {//OPENING WHILE LOOP
        d=n%10;
        for(int i=1;i<=d;i++)//APPLYING CONDITIONS
        {//OPENING FOR LOOP
            p=p*i;
        }//CLOSING FOR LOOP
        sum=sum+p;
        n=n/10;
        p=1;
    }//CLOSING WHILE LOOP
    if(num==sum)//CHECKING WHETHER STRONG NUMBER OR NOT
    cout<<num<<" STRONG NUMBER";
    else
    cout<<num<<" is not STRONG NUMBER";
     return 0;
}//closing main

File: Day-12/program2.java
import java.util.Scanner;
public class program2 
{//opening class
    static boolean arm(int a) //FUNCTION TO CHECK WHETHER ARMSTRONG OR NOT
    {
        int num,d=0,sum=0,r;//initialising variables
        num=a;
        while(num!=0)
        {
            num=num/10;
            d++;
        }
        num=a;
    while (num!=0)
    {//opening while loop
        r=num%10;
        sum =sum+(int)Math.pow(r, d);
        num = num / 10;
    }//closing while loop
    if (sum == a)//applying conditions for armstrong number
    return true;
    else
    return false;
}
    public static void main(String[] args) 
    {//opening main
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter number: ");
        int n= sc.nextInt();
        System.out.println(arm(n));//calling function
        sc.close();
    }//closing main
}//closing class



    

